src/constants.ts

```diff
@@ -6,7 +6,9 @@ export enum Inputs {
 }
 
 export enum Outputs {
-    CacheHit = "cache-hit"
+    CacheHit = "cache-hit",
+    Key = "key",
+    MatchedKey = "matched-key"
```

Comment:
Why are these two needed?

Reply:
So that they can be outputted from restore-only action and reused in the save action. 🤔